Users/abehenderson/CSX/Callbacks/groupBy.js

```diff
@@ -0,0 +1,65 @@
+// Attempt #2 
+  // Per Cole's Change Request:
+    // Initialize a variable equal to invoking the callback on 'el'
+    // Eliminate 'cache' and use '[el]' to temporarily store values
+
+function groupBy(arrays, callback) {
+  const obj = {};
+  arrays.forEach(el => {
+    let evRes = callback(el); // Cole: what is a better name that 'evRes'?
```

Comment:
how about `key`? Fundamentally that's what you're creating: the keys of this object, which are the returned values of `callback(el)`.

Reply:
Ok, of course!  That seems rather obvious now that you mention it.